src/category_theory/category/Rel.lean

```diff
@@ -10,6 +10,7 @@ import category_theory.category.basic
 > https://github.com/leanprover-community/mathlib4/pull/822
 > Any changes to this file require a corresponding PR to mathlib4.
 
+
```

Comment:
@semorrison Do you have any idea why the bot is adding this empty line? Because L12 is already an empty line.

Reply:
Oh no, I thought I'd fixed that

Reply:
```suggestion
```